SimpleMvvmToolkit.Samples.Messaging.Core/SimpleMvvmToolkit.Samples.Messaging.Core/SimpleMvvmToolkit.Samples.Messaging.Core.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="SimpleMvvmToolkit.Express" Version="2.0.0" />
+    <PackageReference Include="System.Windows.Interactivity.WPF" Version="2.0.20525" />
```

Comment:
Remove the reference to **System.Windows.Interactivity.WPF** and add package references to **Microsoft.Extensions.DependencyInjection** and **Microsoft.Xaml.Behaviors.Wpf**, as in the [SimpleMvvmToolkit.Samples.WPF.Core.csproj](https://github.com/SimpleMvvm/SimpleMvvmToolkit.Samples.WPF.Core/blob/master/SimpleMvvmToolkit.Samples.WPF.Core/SimpleMvvmToolkit.Samples.WPF.Core.csproj) file in **SimpleMvvmToolkit.Samples.WPF.Core**.